Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportApiHandler.cs

```diff
@@ -362,5 +368,65 @@ private async Task<bool> SendRestRequestAndGetResponse(string api, AccountReport
                 return false;
             }
         }
+
+        public List<AccountReportBusinessFlow> GetBusinessflowExecutionDataToCentralDB(Guid executionId)
```

Comment:
If we are getting data from Central DB, please change function name to have 'From' instead of 'To'.